### PR TITLE
Feat:Added SignUp Page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added SignUp Page

Screenshot from 2022-03-27 19-35-34

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to go

File: src/Pages/SignUpPage/SignUpPage.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Navbar } from '../../Components/Navigation/Navbar'
+import { SignUp } from '../../Components/SignUp/SignUp'
+
+const SignUpPage = () => {
+  return (
+    <div>
+        <Navbar/>
```
Comment (harshitadaswani) [inline src/Pages/SignUpPage/SignUpPage.js:8]: instead of reusing the component again and again you can use this component directly in app.js

**Review by harshitadaswani — APPROVED** [review]
The UI looks good!! Also, the breakdown of components is nice!!

File: src/Pages/SignUpPage/SignUpPage.js (hunk 1 above)
Comment (neetukumari4858) [reply src/Pages/SignUpPage/SignUpPage.js:8]: okey

File: src/Components/SignUp/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    <input type= "text" className="user-input" placeholder="Enter your Last Name"/><br></br><br></br>
+
+                    <label>Email address</label><br></br>
+                    <input type= "text" className="user-input" placeholder="annu@neog.com"/><br></br><br></br>
+
+                    <label>Password</label><br></br>
+                    <input type= "text" className="user-input" placeholder="..................."/><br></br><br></br>
+
+                    <label>Confirm Password</label><br></br>
+                    <input type= "text" className="user-input" placeholder="..................."/><br></br>
```
Comment (tushartiwari7) [inline src/Components/SignUp/SignUp.js:22]: Use margins and padding to add spacing instead of br tag.

**Review by tushartiwari7 — APPROVED** [review]
Added a comment, please have a look.

**Pull request merged**